src/server/config.rs

```diff
@@ -256,10 +256,10 @@ impl Default for Config {
             status_thread_pool_size: 1,
             max_grpc_send_msg_len: DEFAULT_MAX_GRPC_SEND_MSG_LEN,
             raft_client_grpc_send_msg_buffer: 512 * 1024,
-            raft_client_queue_size: 8192,
+            raft_client_queue_size: 16384,
```

Comment:
nit: please add some annotations here to specify that the default value has been enlarged by this pr.

Reply:
In what situations would we care when this default value is increased, requiring us to look this PR?

Reply:
If there are no such situations, we don't need an annotation here.

Reply:
Better to leave this note as a reminder.

Reply:
okay

Reply:
done